.github/workflows/main.yml

```diff
@@ -138,6 +138,67 @@ jobs:
           --runtime ${{ matrix.rid }}
           --self-contained
 
+      - name: Set permissions
+        if: ${{ !startsWith(matrix.rid, 'win-') }}
+        run: chmod +x ${{ matrix.app }}/bin/publish/${{ matrix.asset }}
+
+      - name: Generate macOS .app bundle resources
+        if: matrix.app == 'DiscordChatExporter.Gui' && startsWith(matrix.rid, 'osx-')
```

Comment:
Do we not also want to do that for the CLI?

Reply:
`.app` bundles are only for graphical macOS applications 
CLI programs are run as executable files like on Linux, such as in `./DiscordChatExporter.Cli`